content/en/docs/collector/custom-collector.md

```diff
@@ -133,7 +133,7 @@ All you need now is to let the `ocb` do it's job, so go to your terminal and
 type the following command:
 
 ```cmd
-ocb --config builder-config.yaml
+./ocb --config builder-config.yaml
```

Comment:
Is this path prefix necessary?

Reply:
`ocb` is an executable file. Without this prefix, an error will occur, so I think it's necessary.

Reply:
Adding a path prefix like `./` would only be necessary if you are trying to force the use of a local executable `./obc` vs. any `obc` command that might be picked up from your PATH.

I haven't stepped through the instructions in this page, but IMHO, they probably work without the `./` path prefix.

Thoughts @cartermp et all?

Reply:
Mr. Chalin, thanks for your patient reply! I'm just a rookie, so it's up to you to decide whether it's necessary or not. At the same time, I am very happy to learn something from your reply.😄

Reply:
If it's not added to the PATH by default, then we should either document it this way or state that you should add it to the PATH. Compare this with other docs with other executables (e.g., package managers) where the executable is automatically ont he path.

Reply:
I just went back and re-read from the start of the page and saw that the command is locally installed (via step 1). Appending `./` to the command path is the simplest solution. So this is good to go!